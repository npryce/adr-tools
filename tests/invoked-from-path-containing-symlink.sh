ln -s ../../../src/adr adr
./adr | head -1
