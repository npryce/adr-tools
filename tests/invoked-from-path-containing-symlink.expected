ln -s ../../../src/adr adr
./adr | head -1
usage: adr help COMMAND [ARG] ...
